src/js/themes/base.js

```diff
@@ -937,6 +937,13 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       },
       margin: { bottom: 'small' },
       // round: undefined,
+      survey: {
```

Comment:
I'm pausing on the naming and alignment with our Figma direction of calling it "Feedback" -- is this a spot where we are okay naming the Grommet thing differently than the Figma thing? Open to discussion, just want to prompt the topic.

Reply:
honestly would this even be in here? I put it so we can test it out but should this be a default for grommet or just have Feedback in our hpe theme

Reply:
I'm inclined to use "survey" in the base grommet theme.

Reply:
I'm OK with the naming being broader than "feedback" on the grommet side. I think it aligns better with what it is for generally.